src/modules/Functions.ts

```diff
@@ -1,20 +1,28 @@
-import { Guild, MessageEmbed, TextChannel } from "discord.js";
+import {
+  Guild,
+  MessageAttachment,
+  MessageEmbed,
+  PermissionOverwrites,
```

Comment:
```suggestion
  MessageEmbed,
```

`PermissionOverwrites` is defined but not used.